docs/docs/migration-guide.mdx

```diff
@@ -24,6 +26,759 @@ If you still have training data in Markdown format then the recommended approach
 to convert your data from Markdown to YAML. Please use the commands described
 [here](./migration-guide.mdx#training-data-files).
 
+### Custom Policies and Custom Components
```

Comment:
Structure comments:
- The toggle-able examples are awesome 
- Content of each section is already structured more or less as "Before...", "Now...", make that consistent 
- Would suggest grouping subheadings according to "new" - things which have no equivalent in 2.0, like `Registering your Component` vs. "changes" - e.g. `Instantiating a Policy for Training`. 
Could just be by adding  a label like `NEW: <subheading>` for the new things.

- Could put the `Registering your <>` guide for both components & policies together, since it is  new for both and is covered together in the guide.

- Order of items (somewhat a matter of opinion) but I'd suggest doing the following for both policies & components:
   - Instantiating a <> for Training
   - Persisting a Trained <>
   - Instantiation a Trained <> 
   - Providing a Default Configuration for a <>
   ... other sections ...

Reply:
> Would suggest grouping subheadings according to "new" - things which have no equivalent in 2.0, like Registering your Component vs. "changes" - e.g. Instantiating a Policy for Training.
Could just be by adding a label like NEW: <subheading> for the new things.

I found this somewhat hard because somewhat everything is conceptually a `CHANGE` for me. `NEW` makes it sounds like something which wasn't before and which they now *can* do. However, they *have* to do the `NEW` things to make things work.

> Could put the Registering your <> guide for both components & policies together, since it is new for both and is covered together in the guide.

I kept it separate put things more into context and also to have full independent step by step guides. Happy to adapt in case you think it makes sense

Reply:
What do you mean with the `Before` / `Now` ones and making this more consistent?

Reply:
E.g. this has a "no longer"... "instead"... so it's obvious what was the situation and what it is now.
```
Policies are no longer instantiated via their constructor. Instead, all policies have
to implement a `create` method.
```
This one goes the other way around:
```
Policies are responsible for persisting themselves. Previously the `persist` method
was called from the outside.
```
I'm suggesting make them all like the first example (not the wording, just the order).

Reply:
done!